### DISCUSSION TITLE
Start a laravel 11 app with the minimum number of files

### DISCUSSION THREAD

**Discussion by dp-anto** [laravel/framework#53332] [Q&A]
I'm trying to start a Laravel 11 app with the minumum number as possible of files. 

According to [this](https://laravel-news.com/laravel-11-directory-structure) and [this](https://rezakhademix.medium.com/laravel-11-no-http-kernel-no-casts-no-console-kernel-721c62adb6ef), Laravel 11 should set an empty `config` directory when creating a fresh new project. However when I execute `laravel new example-app` the `config` directory pullulates of config files.
![2024-10-29_13-29](https://github.com/user-attachments/assets/1cf0425b-ecf5-4f1b-b2a1-987e7bb1dfd3)

Is there any option/flag/command that I'm missing?

I'm running `Laravel Installer 5.9.2` with `Laravel Framework 11.29.0`.

**Comment by noefleury** [comment]
Looking at what the `config` folder contain at the date of the video, **it was indeed totally empty** : https://github.com/laravel/laravel/tree/dc4e89b652548491907fc5b61fe01924b6ea6a5e/config due to the slim skeleton first changes : https://github.com/laravel/laravel/commit/428a190050866c53043900dee91c579b0fa40768

It looks like some files came back on 23 February : https://github.com/laravel/laravel/commit/96508d43ec89ef08c30d155f8e3e81be9f52b17e

And I think you shouldn't care too much about that as some things could be broken when these files are missing.

**Reply by dp-anto** [reply]
@noefleury So should I leave all config files? Because I'm creating a simple test app where I don't need any of them, except `app.php`.

**Comment by Chrisx84** [comment]
isnt lumens still a thing? thats a waterdowned version of laravel, or use to be.

**Reply by noefleury** [reply]
> we no longer support beginning new projects with Lumen

https://lumen.laravel.com/docs/11.x
